airflow/providers/celery/executors/default_celery.py

```diff
@@ -72,7 +72,7 @@ def _broker_supports_visibility_timeout(url):
     "accept_content": ["json"],
     "event_serializer": "json",
     "worker_prefetch_multiplier": conf.getint("celery", "worker_prefetch_multiplier", fallback=1),
-    "task_acks_late": True,
+    "task_acks_late":  conf.getboolean("celery", "task_acks_late", fallback=False),
```

Comment:
We need to keep the default value True
```suggestion
    "task_acks_late":  conf.getboolean("celery", "task_acks_late", fallback=True),
```